.github/workflows/ci.yml

```diff
@@ -129,14 +127,12 @@ jobs:
 
       - name: "Setup DiscoPoP Profiler - Build"
         run: |
+          rm -rf build
           mkdir build
```

Comment:
I have no idea why, but for some reason `mkdir build` complained about it already existing.